### PR TITLE
README.md modifications for Starter Kit workflow

### PR DESCRIPTION
**Pull request by christiancompton**

### CODE REVIEW

File: ios_swift/README.md [hunk 1]
```diff
@@ -106,31 +135,25 @@ curl -H "Content-Type: application/json" -X POST -d $data https://$ACCOUNT.cloud
...
 > See [`ResumeAR/schema.json`](ResumeAR/schema.json) for additional information about the Cloudant database configuration.
 
-3. Run the app and point the camera view to your image.
-
-# Sample Output
-
-![Sample Output](../images/sanjeev_sample_output.png)
-![Sample Output](../images/steve_sample_output.png)
-![Sample Output](../images/scott_sample_output.png)
+1. Run the app and point the camera view to your image.
```
Comment (sanjeevghimire) [inline ios_swift/README.md]: this should be 3. i think

**Comment by stevemar** [comment]
@christiancompton is there some sort of template that you can point us to? we've been using the same flow that you see here for most of our assets, they mostly match up, but just want to see where we're not matching up.

**Comment by christiancompton** [comment]
@stevemart Most follow this format, but it is negotiable.

https://github.ibm.com/Pluto/starter-weather/tree/development/ios_swift

### Sections
+ Title
+ TOC
+ Summary - what is this app, high level, why is this useful
+ Requisites - Need to make it very explicit this app will not run in simulator
+ Configuration - same as your `steps`
+ Run - how do I know this works, how do I use it to do what it says it does, what do I see when it does not work,  how do I extend this?
+ Resources (if applicable)
+ License

The Parent README works really well for a Code Pattern.  It is far too verbose and loosely structured for a starterkit, where everything comes preconfigured, and it works out of the box.

**Review by sanjeevghimire — COMMENTED** [review]
@stevemart there are 2 readme's here. one inside `ios_swift` folder. the one in parent folder is according to our code pattern template. @christiancompton is changing the one inside ios_swift folder as thats the folder users download from starter kit page.

File: ios_swift/README.md (hunk 1 above)
Comment (christiancompton) [reply ios_swift/README.md]: Fixed!

**Review by stevemar — APPROVED** [review]
i'm OK with this.

**Pull request merged**